views/profile.pug

```diff
@@ -0,0 +1,18 @@
+extends layout.pug
+
+block content
+  -
+    let heading;
+    if (theUser)
+      heading = theUser.displayName
+    else
+      heading = "No user found"
+  .card
+    .card-body
+      h1.card-title= heading
```

Comment:
This can be a 1liner:
```pug
h1.card-title= theUser.displayName || "No user found"
```
But, if the user is not found, it should send a 404 page, not this page

Reply:
it's a relic of the old system. no user = 404